newsfragments/47264.significant.rst

```diff
@@ -0,0 +1,26 @@
+Removed leftover deprecations prior to 3.0.0.
+
+* Removed the ``RemovedInAirflow3Warning`` warning class.
+* Removed the deprecated module ``airflow.api.auth.backend.session``. Please use ``airflow.providers.fab.auth_manager.api.auth.backend.session`` instead.
+* Removed the deprecated ``cleanup_stuck_queued_tasks`` method from the ``BaseExecutor`` interface. It is replaced by function ``revoke_task``.
+
+* Types of change
+
+  * [ ] Dag changes
+  * [ ] Config changes
+  * [ ] API changes
+  * [ ] CLI changes
+  * [ ] Behaviour changes
+  * [ ] Plugin changes
+  * [ ] Dependency changes
+  * [x] Code interface changes
+
+
+dict[str, dict[str, Collection[str]]]
+None
+
+!=
+
+dict[str, dict[str, Collection[str]]]
+dict[str, Collection[str]]
+None
```

Comment:
Should that be removed ?

Reply:
Ups, how did this end-up here :-D Thanks for finding!